.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: c
+sudo: required
+dist: trusty
+compiler:
+  - gcc
+os: linux
+before_install:
+- sudo apt-get -qq update
+- sudo apt-get install -y dpkg-dev debhelper dkms lintian
+- sudo apt-get install linux-headers-`uname -r`
+- sudo cp -R . /usr/src/digimend-7
```

Comment:
Could you try replacing this with `git checkout-index --all --prefix /usr/src/digimend-7`? This way we won't copy the whole `.git` directory with the history there. Also, could you perhaps move this to the `script` section? It feels more a part of the tests.